LunaMary/src/Main.java

```diff
@@ -1,14 +1,155 @@
+    }
 
     public static void main(String[] args) {
 
-        Menu menu = new Menu();
-        menu.registerPerson();
-        menu.showAsks();
+        TypeQuiz typeQuizPsic = new TypeQuiz("Psicologico");
+        TypeQuiz typeQuizMath = new TypeQuiz("Matematico");
+
+
+        Option option1 = new Option("Haces pasar a una chica que crees que baja después que tú…", 10, Sigla.A, false);
+        Option option2 = new Option("Te pones en un rincón", 10, Sigla.B, false);
+
+        Option option3 = new Option("Llamas a emergencias y procuras que la gente no se le agolpe", 10, Sigla.A, false);
+        Option option4 = new Option("Le preguntas lo que ha pasado", 10, Sigla.B, false);
+
+
+        List<Option> optionList1 = new ArrayList<Option>();
+        optionList1.add(option1);
+        optionList1.add(option2);
+
+        List<Option> optionList2 = new ArrayList<Option>();
+        optionList2.add(option3);
+        optionList2.add(option4);
+
+
+        SelectedQuiz selectedQuiz1 = new SelectedQuiz("COMO TE COMPORTAS CUANDO ENTRAS EN UN ASCENSOR?", optionList1);
+        SelectedQuiz selectedQuiz2 = new SelectedQuiz("COMO TE COMPORTAS CUANDO TE ENCUENTRAS CON UNA PERSONA QUE SE HA ACCIDENTADO EN LA CALLE?", optionList2);
+
+        LiteralQuiz literalQuiz = new LiteralQuiz("Que opinas sobre la tecnologia");
+
+        List<IQuestion> iQuestions1 = new ArrayList<IQuestion>();
+        iQuestions1.add(selectedQuiz1);
+
+        List<IQuestion> iQuestions2 = new ArrayList<IQuestion>();
+        iQuestions2.add(selectedQuiz2);
+
+        Quiz quiz1 = new Quiz(iQuestions1, typeQuizPsic);
+        Quiz quiz2 = new Quiz(iQuestions2, typeQuizPsic);
+
+        List<Quiz> quizList = new ArrayList<Quiz>();
+        quizList.add(quiz1);
+        quizList.add(quiz2);
+
+        List<AnswerPerson> answerPeople = new ArrayList<AnswerPerson>();
+
+        QuizLibrary quizLibrary1 = new QuizLibrary(quizList, answerPeople, "Psicologico");
+        QuizLibrary quizLibrary2 = new QuizLibrary(quizList, answerPeople, "Matematico");
+
+        List<QuizLibrary> quizLibraryList = new ArrayList<QuizLibrary>();
+        quizLibraryList.add(quizLibrary1);
+        quizLibraryList.add(quizLibrary2);
+
+
+        //Lista para Personas
+        List<Person> personList = new ArrayList<Person>();
+
+        boolean exit = false;
+        print("Welcome to your Quiz");
+
+        while (!exit) {
 
+            print("Please enter an option");
+            print("1. Register yourself");
+            print("2. Exit");
+            String option = scanner.nextLine();
+            switch (option) {
+                case "1":
+                    registerPerson(personList);
+                    print("The size Person is: " + personList.size());
+                    break;
+                default:
+                    exit = true;
+            }
 
+
+            while (!exit) {
+                print("Please select a new Quiz");
+                print("1. Psicologico");
```

Comment:
The types should be printed dynamically based on the Quizzes that you have available on your QuizLibrary